lib/gettext.nit

```diff
@@ -0,0 +1,80 @@
+# Internationalization of Strings using `gettext` library
+module gettext is
+	new_annotation language
+end
+
+in "C header" `{
```

Comment:
Tu devrais pouvoir mettre ça dans le `in "C"` seulement. Tu as besoin du header que si tu déclare des classes externes.